src/main/java/org/petrax/models/User.java

```diff
@@ -15,46 +15,50 @@ public class User {
     private int id;
 
 
-    @NotBlank(message = "Name is required")
+    @NotBlank(message = "First name is required")
     @Size(min = 3, max = 50, message = "Name must be between 3 and 50 characters")
-    private String name;
+    private String firstName;
 
-    @Size(max = 500, message = "Description too long!")
-    private String description;
+    @NotBlank(message = "Last name is required")
+    @Size(min = 2, max = 50, message = "Last name must be between 2 and 50 characters")
+    private String lastName;
 
     @NotBlank(message = "Email is required")
     @Email(message = "Invalid email. Try again.")
     private String contactEmail;
 
```

Comment:
These fields look great! One quick thing-- do we have a field planned for a username outside of a users' first or last name? A unique user name/ email will be important for database management.

Reply:
Thanks so much! I will get that updated.